xunit-autolabeler-v2/ast_parser/lib/file_utils.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Small nit:

Maybe you can use the same license header? It's slightly different here.